calour/training.py

```diff
@@ -0,0 +1,100 @@
+'''
+filtering (:mod:`calour.filtering`)
+===================================
+
+.. currentmodule:: calour.filtering
+
+Functions
+^^^^^^^^^
+.. autosummary::
+   :toctree: generated
+
+   filter_by_data
+   filter_training
+'''
+
+
+from logging import getLogger
+
+from sklearn.feature_extraction import DictVectorizer
+from scipy.sparse import hstack
+import pandas as pd
+import numpy as np
+
+from .experiment import Experiment
+
+
+@Experiment._record_sig
+def onehot_encode_features(exp, fields, sparse=None, inplace=False):
```

Comment:
is onehot_encode a common term? otherwise, maybe change name to: encode_sample_info_in_table()? or encode_onehot_features() ?

Reply:
yes. see http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.OneHotEncoder.html